distributed/utils.py

```diff
@@ -359,12 +359,17 @@ def convert(task):
 
 import logging
 from .compatibility import logging_names
-logging.basicConfig(format='%(name)s - %(levelname)s - %(message)s',
-                    level=logging.INFO)
+
+handler = logging.StreamHandler(sys.stderr)
+handler.setLevel(logging.INFO)
+fmt = logging.Formatter('%(name)s - %(levelname)s - %(message)s')
+handler.setFormatter(fmt)
 
 for name, level in config.get('logging', {}).items():
     LEVEL = logging_names[level.upper()]
-    logging.getLogger(name).setLevel(LEVEL)
+    logger = logging.getLogger(name)
+    logger.setLevel(LEVEL)
+    logger.addHandler(handler)
 
 # http://stackoverflow.com/questions/21234772/python-tornado-disable-logging-to-stderr
 stream = logging.StreamHandler(sys.stderr)
```

Comment:
Logging only critical messages seems heavy-handed. This will even silence exceptions when `add_callback` callbacks are called. I would recommend setting the level to ERROR instead.